cpp/ql/src/Critical/OverflowStatic.ql

```diff
@@ -58,7 +58,7 @@ predicate overflowOffsetInLoop(BufferAccess bufaccess, string msg) {
 }
 
 predicate bufferAndSizeFunction(Function f, int buf, int size) {
-  f.hasGlobalOrStdName("read") and buf = 1 and size = 2
+  f.hasStdName("read") and buf = 1 and size = 2
```

Comment:
What does the `hasStdName()` predicate do exactly?  I ask because I don't believe that `read` (and other unbuffered I/O routines) get pulled into `namespace std`.

Reply:
I did that backwards - fixed after the rebase